src/server/main.py

```diff
@@ -48,7 +49,14 @@ def _setup_endpoints(self) -> None:
 
 def hello_world_endpoint() -> str:
     """Root endpoint."""
-    return "<p>Hello, World!</p>"
+    try:
+        with DatabaseProvider.handler() as handler:
+            handler().execute(QUERIES.HELLO_WORLD_QUERY)
+            (db_hello_world,) = handler().fetchone()  # type: ignore
+    except:
+        # no matter why, show something else than it would show if it worked correctly
+        db_hello_world = "Goodbye, World!"
+    return f"<p>Hello, World!</p><p>{db_hello_world}</p>"
```

Comment:
Imho this should be closed.

Reply:
Niestety, ale testowanie przez logi jest bardziej skomplikowane, szczególnie jeśli ktoś korzysta z konsoli i/lub nie ma vscode. 
W związku z tym proponuję albo to, albo zmienić "warning" na "error" i wysypać kontener w przypadku niepowodzenia.
Gdyż aplikacja jest w stanie cokolwiek robić bez połączenia z bazą danych>

Reply:
VSCode does the same things that you can do in a terminal, it is just hidden under convenient buttons. Let's revise what we can do (and what we have been doing for the past two months):

```bash
# needed anyway, no matter what tool you use
export CTB_DB_NAME=...
export CTB_DB_USER=...
export CTB_DB_PWD=...
export CTB_DB_HOST=...

# local launch
python -m flask --app ./src/server/main.py run

# docker launch
docker build --tag ctb-backend .
docker run -it -e CTB_DB_NAME -e CTB_DB_USER -e CTB_DB_PWD -e CTB_DB_HOST ctb-backend  # manually
docker run -it --env-file ./.env ctb-backend  # from .env file
```

Whatever way of launching You may choose, logs will tell you whether Your configuration is correct or not (and by extension, _"is the database working or not"_), i.e.:

```
ERROR: connection failed: password authentication failed for user ...
FATAL:  password authentication failed for user ...
WARNING: cannot connect to a file database: UNKNOWN_ERROR(1)
```

By the way, as you might have noticed, I used the `.env` file. We already have an exhausting list of possible envfile signatures in our [`.gitignore`](https://github.com/PZ-CTB/ctb-backend/blob/f6f3e56d3957bcc55be0b3e30ec213ec52ca6bab/.gitignore#L104-L111). Feel free to choose one of them instead of `env.list`.

So to once again sum everything up, I don't think this addition is necessary.

Reply:
Tyle że nikogo nie będzie obchodzić, co było tu robione i jak było odpalane, instrukcja ma być przygotowana tak, by zadziałała wszędzie bez żadnych dodatkowych konsultacji z nami. I przy okazji możliwie jak najszybciej. Tak samo trzeba podkreślić, że na serwerach produkcyjnych często nie mamy dostępu do vscode, gdyż na nie instaluje się tam środowiska deweloperskiego. Pozostaje tylko konsola. Natomiast przeglądanie logów dockera z poziomu konsoli jest zdecydowanie bardziej czasochłonne, niż wejście przez przeglądarkę, czy tam kliknięcie w niej F5. A jak to mówią: czas to pieniądz.
Plik env.list był w dokumentacji dockera, jednak nie widzę przeciwwskazań, aby nazywał się np. .env, skoro to już jest w gitignore.

Reply:
I don't know why do you keep mentioning VSCode, since it is completely unrelated to what this change adds and to the problem of testing database connection. You have to launch the backend somehow anyway, so you are already in the right place to check the logs if database works or not.

The concept of checking the database connection through an endpoint is just wrong to me (and it's actually slower than reading one line of log displayed right under your launch command).

Regarding developer environment - it's a separate topic, and yes, we should mention in wikipedia that we support VSCode through tasks and launch configurations.

Anyways, this PR must not be labeled under CTB-137, because that task is related only to documentation, not debugging improvements. Please close it.